src/functions-templates/js/slack-rate-limit/slack-rate-limit.js

```diff
@@ -38,12 +39,11 @@ class IdentityAPI {
 
       if (!response.ok) {
         return response.text().then(data => {
-          return Promise.reject({ stauts: response.status, data })
```

Comment:
Looks like there was a typo there `stauts`

Reply:
Yes, these errors are handled here:
https://github.com/netlify/cli/blob/6916ba0ae3d2a142965fd3eade565c58e5f206e0/src/functions-templates/js/slack-rate-limit/slack-rate-limit.js#L120
https://github.com/netlify/cli/blob/6916ba0ae3d2a142965fd3eade565c58e5f206e0/src/functions-templates/js/slack-rate-limit/slack-rate-limit.js#L124

So the thrown objects were implicitly stringified.